actions/service_test.go

```diff
@@ -155,4 +161,10 @@ hooks:
 	if runResult != nil {
 		t.Errorf("GetRunResult() result=%v, expected nil", runResult)
 	}
+
+	bytes, err := ioutil.ReadAll(outputReader)
+	require.NoError(t, err)
+	str := string(bytes)
+	rows := strings.Split(str, "\n")
+	require.Greater(t, len(rows), 10)
```

Comment:
You can use bytes.Split if you just want to count rows, and prevent the string()